tests/trezor/trezor_tests.cpp

```diff
@@ -1245,10 +1263,14 @@ void gen_trezor_base::set_hard_fork(uint8_t hf)
   m_top_hard_fork = hf;
   if (hf < 9){
     throw std::runtime_error("Minimal supported Hardfork is 9");
-  } else if (hf == 9){
+  } else if (hf <= 11){
     rct_config({rct::RangeProofPaddedBulletproof, 1});
-  } else {
+  } else if (hf == 12){
     rct_config({rct::RangeProofPaddedBulletproof, 2});
+  } else if (hf == 13){
```

Comment:
There are HF_VERSION_* defines for those if needed.

Reply:
fixed